assets/Icons/README.md

```diff
@@ -1,44 +1,17 @@
 # Source SVG icons used to build React components
 
-### Here we collect the raw (exported from Sketch) and optimized SVGs.
+## To add a new icon or icons
 
-1. Add raw SVGs to assets/Icons/raw folder
+1. Add prepared SVGs to assets/Icons/optimized folder
 
-2. Process SVGs with svgo
+2. Run `yarn create:svg:components`
 
-If necessary, install svgo. `npm install svgo -g`
-
-```
-cd assets/Icons
-svgo --pretty --indent=2 (remove title and fill) raw/ -o optimized/
-```
-(Exported SVGs require the `<title>` tag and the "fill" attribute removed.)
-
-3. Make a React component using this template (we are working on a tool to do this):
-
-```
-(Example.tsx)
-/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
-/* tslint:disable:max-line-length */
-import * as React from 'react';
-
-const Example = function Example(props: any) {
-  return (
-    <svg width="240" height="240" viewBox="0 0 240 240" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" {...props}>
-      <path fillRule="evenodd" d="xxx"/>
-    </svg>
-  );
-};
-
-export default Example;
-```
-
-Next, add the component file you created to src/components/Icon/icons/ and continue the steps for [Adding Icons](https://github.com/Microsoft/YamUI#adding-icons)
+3. Export the new icon from src/components/Icon/icons/index.ts
 
 4. Commit all these resources to a new branch.
 
 ### SVG requirements
  - square field of 240
- - includes viewBox, width and height
+ - viewBox required, height and width optional
  - no id or fill in path element. fill-rule ok.
```

Comment:
if we use svgo to clean the SVGs we import you can delete line 16 as id and fill will be removed